test/unit/test_helpers.js

```diff
@@ -0,0 +1,101 @@
+import _ from 'lodash';
+import { check, property, gen } from 'testcheck';
```

Comment:
I've been putting them in test_helpers.js when multiple files need them. Otherwise, I just define them above where they're needed in the same file.